### PR TITLE
Update OktaLogger pod spec

### PR DESCRIPTION
**Pull request by artemshvydkyi-okta**
Update OktaLogger pod spec

### CODE REVIEW

**Review by IldarAbdullin-okta — CHANGES_REQUESTED** [review]
@sandeeppenchala-okta , @artemshvydkyi-okta , I don't understand the change. What we are fixing?

**Comment by artemshvydkyi-okta** [comment]
@IldarAbdullin-okta Podspec had incorrect dependencies. Version with incorrect dependencies didn't provide to launch unit tests of the extensions in the main app.

File: OktaLogger.podspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.subspec "Complete" do |complete|
     complete.dependency 'OktaLogger/FileLogger'
     complete.dependency 'OktaLogger/FirebaseCrashlytics'
-    complete.ios.dependency 'OktaLogger/InstabugLogger'
+    complete.dependency 'OktaLogger/InstabugLogger'
```
Comment (sandeeppenchala-okta) [inline OktaLogger.podspec:20]: do we need this change?, we only support for iOS

Comment (artemshvydkyi-okta) [reply OktaLogger.podspec:20]: Without this change `pod lib lint` validation doesn't pass

Comment (sandeeppenchala-okta) [reply OktaLogger.podspec:20]: This might fail when you integrate in OV, please test it locally. macOS don't have Instabug support

**Review by sandeeppenchala-okta — APPROVED** [review]

File: OktaLogger.podspec (hunk 1 above)
Comment (artemshvydkyi-okta) [reply OktaLogger.podspec:20]: Double checked and everything is good

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM!

**Pull request merged**